src/nvim/math.h

```diff
@@ -9,6 +9,18 @@ static inline bool is_power_of_two(uint64_t x)
   return x != 0 && ((x & (x - 1)) == 0);
 }
 
+// Function version of MIN to avoid evaluating expressions twice.
+static inline int min_i(int x, int y)
```

Comment:
I've seen `minimum` and `maximum` as alternative names to avoid collision with `min`/`max`. (I think `min_i` is a bit too opaque.)

Out of interest: Some compilers are case-sensitive, so _could_ we just name them `MIN` and `MAX` (even though they're not "really" macros)?

Reply:
We already have `MIN` and `MAX` and need to maintain them for non-`int` types.

Reply:
Then `min_int` etc. would be nicer than `min_i`, IMO.